Cannot find module '../lib' from 'src/web/node_modules/react-icons/fa/index.js'

After landing @DukeManh's recent PR, we're failing on `master` with:

```
FAIL src/web/src/components/AboutFooter.test.tsx
  ● Test suite failed to run

    Cannot find module '../lib' from 'src/web/node_modules/react-icons/fa/index.js'

    Require stack:
      src/web/node_modules/react-icons/fa/index.js
      src/web/src/components/AboutFooter.tsx
      src/web/src/components/AboutFooter.test.tsx

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:306:11)
      at Object.<anonymous> (src/web/node_modules/react-icons/fa/index.js:2:15)

  console.warn
PASS src/web/src/components/Logo.test.tsx
  ✓ renders correctly (23 ms)

    Material-UI: The `css` function is deprecated. Use the `styleFunctionSx` instead.

      at css (src/web/node_modules/@material-ui/system/styleFunctionSx.js:75:13)
      at Object.<anonymous> (src/web/node_modules/@material-ui/core/Box/Box.js:14:37)
      at Object.<anonymous> (src/web/node_modules/@material-ui/core/Box/index.js:21:36)

Summary of all failing tests
FAIL src/web/src/components/AboutFooter.test.tsx
  ● Test suite failed to run

    Cannot find module '../lib' from 'src/web/node_modules/react-icons/fa/index.js'

    Require stack:
      src/web/node_modules/react-icons/fa/index.js
      src/web/src/components/AboutFooter.tsx
      src/web/src/components/AboutFooter.test.tsx

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:306:11)
      at Object.<anonymous> (src/web/node_modules/react-icons/fa/index.js:2:15)
```

I don't think they are related, but I don't understand this either.

--- a/src/web/package.json
+++ b/src/web/package.json
@@ -25,7 +25,7 @@
     "next-pwa": "^5.2.14",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "react-icons": "^4.2.0",
+    "react-icons": "4.2.0",
     "react-use": "^17.2.1",
     "smoothscroll-polyfill": "^0.4.4",
     "swr": "^0.5.5",

